### DISCUSSION TITLE
O3DE Release 22.10.0 Feature List and Release Themes Discussion

### DISCUSSION THREAD

**Discussion by tonybalandiuk** [o3de/o3de#12062] [General]
Let's talk about the 22.10.0 feature list and theme(s) of the release.
For the full features list (so far) see (https://github.com/o3de/sig-release/blob/main/releases/22.10.0/22100%20feature%20list.md ) in this thread. 

First - if anyone has any additional features to add to that list, please go issue a pull request against the file. Alternatively you may post a note here in this discussion for @tonybalandiuk and he will update the feature list.

Second - We need to finalize the theme(s) of the release.

Here is a starting point for the themes- this is where we left off in the Tuesday 9/13 meeting:

Theme(s) of the 2022.10.0 Release - we will go with these 5 unless we receive feedback otherwise:
* Onboarding and Collaboration (Easier to onboard and collaborate with other team members with remote projects, templates, networked asset cache )
* Multiplayer is easier to setup (client-server connection, debugging, network spawning )
* Artist workflow (animation import, root motion extraction, motion matching now experimental)
* Usability improvements (Viewport entity selection)
* Other notable items (Sky Atmosphere, Stars, Terrain performance)

**Comment by rainbj** [comment]
Hi @tonybalandiuk,
I think you are missing a couple items.
- "Usability improvements (Viewport entity selection)." <- [from above] 
  - A sibling update is the prefab edit mode. (black and white mode when editing). The one above talks about selection around all thing when selected. 100% important. However, you also have an additional visualization for prefabs. Now when you double-click on a prefab you now go into prefab edit mode.
  - ![2022-09-19_15-38-59](https://user-images.githubusercontent.com/82551958/191101884-725fcebe-07e6-431b-9082-ebd6b013e563.png)
- Asset Browser updates: I don't see this listed anywhere on your full feature list. Asset Browser project is releasing its phase 1 of 4 phases. 
 - I will send you a link to the project page. However, here are a list of some of the features coming out. 
 - Previously, Asset Browser was just a view of your asset with no way of interacting with your assets. We have now added in some core features you would expect in most asset browser including, duplicating, deleting, open in associated application (material editor if it's a material file, Script canvas if it's a script canvas file, etc.) We also detect allow you to open up asset in 3rd party apps. JSON to Visual studios (or default coding tool), 3d asset in your associated DCC tool, etc. We also now have a create new workflow for the creation of new assets you want to start in a specific folder or directory. The timeline for our 4 phase is not in sync with the release schedule so we are still about to wrap up phase 1.  The developer owner of this project is @lsemp3d. Thanks
 
![2022-09-19_15-35-53](https://user-images.githubusercontent.com/82551958/191104370-fa0c7ebb-7675-4ffd-8f8a-e7745a05d786.png)
![2022-09-19_15-36-17](https://user-images.githubusercontent.com/82551958/191104428-b15920bc-4f7b-44eb-a3ba-6d6a6b99dabd.png)
![2022-09-19_15-37-53](https://user-images.githubusercontent.com/82551958/191104453-854fd13f-00e6-490e-b9c3-300d74714cf7.png)
![2022-09-19_15-47-48](https://user-images.githubusercontent.com/82551958/191104495-6f4aa0ed-15b1-4398-8615-39205545e9e3.png)

**Comment by amzn-leenguy** [comment]
For the theme discussion, i would like to suggest each release could be named after a type of wood. (paying homage to Lumberyard) 

Types of Wood

1. Akasa Wood
2. Ash Wood
3. Balsa Wood
4. Bamboo Wood
5. Beech Wood
6. Brazilian Mahogany Wood
7. Cedar Wood
8. Celtis Wood
9. Conifer Wood
10. Cherry Wood
11. Dahoma Wood
12. Danta Wood
13. Douglas fir Wood
14. European Beech
15. Elm Wood
16. Greenheart Wood
17. Iroko Wood
18. Khaya Wood
19. Ligneous Wood
20. Maple Wood
21. Meranti Wood
22. Oak Wood
23. Oak, European Wood
24. Opepe Wood
25. Okoume Wood
26. Pine Wood
27. Pitch Pine Wood
28. Poplar Wood
29. Redwood, European
30. Sapele Wood
31. Sitka Spruce Wood
32. Southern Yellow Pine
33. Teak Wood
34. Utile Wood
35. Walnut Wood
36. Western Hemlock
37. Western Red Cedar
38. Whitewood, European

**Reply by tonybalandiuk** [reply]
by "theme(s)" of the release I meant things like "Onboarding and Collaboration" or "Artist Workflows" .  Thank you for your suggestions @amzn-leenguy but we're not at the point of having themed names for the release. That proposal was discussed here https://github.com/o3de/sig-release/issues/20 but it was decided to hold off on that for 2022.

**Reply by KylerSF** [reply]
Its good that themed names for releases is held off right now, but I do like this idea a lot of paying homage to lumberyard by using wood names. I feel this should be brought back up in the future when themed names for releases is/might be added. Great idea @amzn-leenguy

**Comment by yuyihsu** [comment]
@rainbj @tonybalandiuk Josh's update on the viewport interaction model can be identified as another theme (Viewport interaction improvement) to highlight, including the object outline state, prefab edit mode, viewport edit mode, etc. @hultonha what other viewport related improvements we will release with this release?

**Comment by yuyihsu** [comment]
One other theme I can think of is our improvements on **editor extension**. Now users can easily extend the engine with a gem creation wizard, and component with an easy to follow [script](https://www.o3de.org/docs/user-guide/programming/components/create-component/).

**Comment by yuyihsu** [comment]
Another theme is our improvements on the asset pipeline improvements (@lmbr-pip might be able to provide more), this includes hot reload, procprefabs, and @Starr's DCCsi tool that can help user import their assets much smoother. @shawstar , can you provide more background and link for this one?

**Comment by Ulrick28** [comment]
We should add Create a Gem Wizard (Wyzard?) under the usability improvements theme. I also believe the asset processor improvements would fall under usability as well. Today, during the TSC meeting Royal made it clear usability is a perceived issue with O3DE, therefore any callouts towards the usability theme could help with this perception. In fact, it might be a good idea for marketing to devote some bandwidth towards hyping the usability improvements.

**Comment by sptramer** [comment]
I think it's important to continue to emphasize work on the following areas (hat tip to @tjmichaels for some):

* *Quality of Life* - This covers a lot of things! Stability, performance, UX and workflow improvements, and better tools for developers to work with O3DE. Some specific features to call out here: Prefab workflow changes, Project Manager improvements, and tools to more easily create Gems and Components.
* *New terrain in preview* - The new O3DE Terrain system is in preview. For developers who have found O3DE best for working in smaller or more enclosed environments, now you can start working on larger-scale worlds with improved terrain.

**Comment by chanmosq** [comment]
The release notes are now in a PR: https://github.com/o3de/o3de.org/pull/2021
Please review and leave comments regarding any changes!
